projects/fsastorefrontlib/src/cms-components/my-account/inbox/unread-messages-indicator/unread-messages-indicator.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="message-indicator-container">
+  <a
+    [routerLink]="
+      {
+        cxRoute: 'inbox'
+      } | cxUrl
+    "
+    class="messages-indicator"
+    *ngIf="user$ | async"
+  >
+    <div class="fs-icon icon-envelope"></div>
+    <ng-container *ngIf="numberOfUnreadMessages$ | async as numberOfMessages">
+      <span
+        *ngIf="numberOfMessages <= 9"
+        class="message-indicator-counter normal-weight"
+        >{{ numberOfMessages }}</span
+      >
+      <span
+        *ngIf="numberOfMessages > 9"
+        class="message-indicator-counter normal-weight higher-indicator"
+      >
+        {{ '9+' }}
```

Comment:
Looks like a good candidate for ng-container with if/else and ng-template for else case.
Perhaps span class can be resolved with if/else as well ```[ngClass]="{'higher-indicator': numberOfMessages > 9}```. 
Also please rethink the use of higher-indicator class, if it's needed at all. Just to add 3px left/right of padding can perhaps be solved differently.

Reply:
Is this acceptable:
`<span
        class="message-indicator-counter normal-weight"
        [ngClass]="{'py-0 px-1' : numberOfUnreadMessages > 9 }"
        >{{ numberOfUnreadMessages > 9 ? '9+' : numberOfUnreadMessages}}
</span>`
This will combine both cases in one, remove higher-indicator class, but still left 9+ interpolated.

Or it's better to use ng-template?

Reply:
Yup, looks good, no need for template then, since you are using it only in one place.